images/tidb-backup-manager/Dockerfile

```diff
@@ -15,6 +15,10 @@ RUN wget -nv http://download.pingcap.org/br-latest-linux-amd64.tar.gz \
     && chmod 755 /usr/local/bin/br \
     && rm -rf br-latest-linux-amd64.tar.gz
 
+RUN curl -sL -o /usr/local/bin/shush \
+  https://github.com/realestate-com-au/shush/releases/download/v1.4.0/shush_linux_amd64 \
```

Comment:
Define an environment variable for the shush version.

Reply:
done